### DISCUSSION TITLE
Production readiness of go-workflows?

### DISCUSSION THREAD

**Discussion by nathan815** [cschleiden/go-workflows#186] [General]
Can go-workflows be considered "production ready" yet? Anyone using it in production right now? I realize it's in pre-1.0. 

This looks like a really nice alternative to DTFx/Temporal. I like that it doesn't need any additional server components as that simplifies deployment/operations especially for multi-tenant use cases.

**Comment by sheltonjynk** [comment]
go-workflows is currently pre-1.0 and not yet considered production-ready. It may be worth exploring, but it's important to carefully evaluate it and weigh the potential risks before using it in a production environment.

**Comment by cloudcompute** [comment]
Hi

Any one using go-workflows in production?

**Reply by cschleiden** [reply]
There are teams who use it in production.

**Comment by lyuboxa** [comment]
✋ yup, to an extent.
